src/pages/groupComparison/Mutations.tsx

```diff
@@ -0,0 +1,206 @@
+                        }
+                        placeholder={'Select a gene'}
+                        loadOptions={this.props.store.loadOptions}
+                        cacheOptions={true}
+                    />
+                </div>
+            );
+        if (this.props.store.selectedMutationMapperGene) {
+            const mutationMapperStore = this.mutationMapperToolStore.getMutationMapperStore(
+                this.props.store.selectedMutationMapperGene!.hugoGeneSymbol
+            );
+            return (
+                <>
+                    {selectorContent}
+                    <div>
+                        <Loader
+                            isLoading={
+                                this.mutationMapperToolStore
+                                    .mutationMapperStores.isPending &&
+                                this.props.store.mutations.isPending
+                            }
+                        />
+                        {mutationMapperStore &&
+                            this.props.store.mutations.isComplete &&
+                            this.props.store.activeGroups.result!.length ===
+                                2 && (
+                                <div>
+                                    <h3>
+                                        {_(
+                                            this.props.store.mutationsByGroup
+                                                .result!
+                                        )
+                                            .keys()
+                                            .slice(0, 2)
+                                            .join(' vs ')}
+                                    </h3>
+                                    <GroupComparisonMutationMapper
+                                        onInit={this.onMutationMapperInit}
+                                        {...convertToMutationMapperProps({
+                                            ...getServerConfig(),
+                                        })}
+                                        generateGenomeNexusHgvsgUrl={hgvsg =>
+                                            getGenomeNexusHgvsgUrl(
+                                                hgvsg,
+                                                undefined
+                                            )
+                                        }
+                                        store={mutationMapperStore}
+                                        showTranscriptDropDown={true}
+                                        plotLollipopTooltipCountInfo={
+                                            this.plotLollipopTooltipCountInfo
+                                        }
+                                    />
+                                </div>
+                            )}
+                    </div>
+                </>
+            );
+        } else {
+            return <>{selectorContent}</>;
```

Comment:
doesn't need react fragment wrapper.

Reply:
<> is shortchut for <React.Fragment> which is only necessary when you have multiple items you want to return together but you have don't have parent